docs/tutorials/using-ev3-leds.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Using the EV3 LEDs
+group: hardware-standard
+---
+
+* Table of Contents
+{:toc}
+
+
+### Basics
+
+When the EV3 is booting, the LEDs flash amber (orange). When it is finished
+booting, they turn solid green. This functionality is hard-coded in the kernel
+and in Brickman.
+
+After the boot is complete, you can control the LEDs yourself. The LEDs live in
+`/sys/class/leds/`.
+
+```
+$ ls /sys/class/leds/
+ev3:left:green:ev3dev  ev3:right:green:ev3dev
+ev3:left:red:ev3dev    ev3:right:red:ev3dev
+```
+
+You probably noticed that _amber_ is missing here. We will get to that in a bit.
+
+Each LED has it's own attributes for controlling it. We are using the standard
+Linux LED device class, so there is a bunch of extra stuff that it not really
+useful. Let's figure out what is...
+
+```
+$ ls /sys/class/leds/ev3:left\:green\:ev3dev
+brightness  device  max_brightness  power subsystem  trigger  uevent
+```
+
+{% include /style/icon.html type="info" %}
+You only have to escape the colons using the backslash (`\:`) when using a command
+line shell. The actual directory name is `ev3:left:green:ev3dev`.
+{: .alert .alert-info}
+
+```
+# turn the left green LED off
+$ echo 0 > /sys/class/leds/ev3:left\:green\:ev3dev/brightness
+# find out what the maximum brightness value is
+$ cat /sys/class/leds/ev3:left\:green\:ev3dev/max_brightness
+255
+# turn the left green LED back on
+$ echo 255 > /sys/class/leds/ev3:left\:green\:ev3dev/brightness
+# dim the left green LED 1/2 way
+$ echo 127 > /sys/class/leds/ev3:left\:green\:ev3dev/brightness
+```
+
+{% include /style/icon.html type="info" %}
+You you must be a member of the `ev3dev` group (or root) to be able to control the LEDs.
+{: .alert .alert-info}
+
+
+### Triggers
+
+Triggers can make the LED do interesting things.
+
+```
+$ cat /sys/class/leds/ev3:left\:green\:ev3dev/trigger
+[none] mmc0 timer heartbeat default-on transient
+legoev3-battery-charging-or-full legoev3-battery-charging legoev3-battery-full
+legoev3-battery-charging-blink-full-solid rfkill0
+```
+
+* `none` means we are manually controlling the LED with `brightness` like we just did.
+* `mmc0` makes the LED blink whenever there is SD card activity. This is what the
+  LEDs were doing during boot.
+* `timer` makes the LED blink. When we change the trigger, we get new attributes
```

Comment:
Qualify the phrase "makes the LED blink" to differentiate from previous line; something like "makes the LED blink periodically".